tests/js/common/shell/shell-index.js

```diff
@@ -55,6 +55,27 @@ function IndexSuite() {
       collection = null;
     },
 
+
+    testIndexProgress : function () {
+      var c = internal.db._create("c0l");
+      var docs = [];
+      for (var i = 0; i < 10000; ++i) {
+        docs.push({name : "name" + i});
+      }
+      c.insert(docs);
+      internal.debugSetFailAt("fillIndex::pause");
+      c.ensureIndex({ type: "persistent", fields:["name"], inBackground: true});
+      //arango.POST("_api/index?collecion=c0l", { type: "persistent", fields:["name"], inBackground: true}, {"x-arango-async": "store"})
+      console.error("hu");
+      require('internal').sleep(1000);
+      console.error("ha");
+      internal.debugSetFailAt("fillIndex::unpause");
+      internal.debugRemoveFailAt("fillIndex::pause");
+      internal.debugRemoveFailAt("fillIndex::unpause");
+      require('internal').sleep(1000);
```

Comment:
While this test may work when running it manually and watching its output, it won't help us in any automated test runs.